src/ps/psreaddata.cpp

```diff
@@ -502,13 +516,22 @@ PetscErrorCode PSReadMatPowerData(PS ps, const char netfile[]) {
           -1; /* xx_end_line points to the next line after the record ends */
   PetscInt gen_start_line = -1, gen_end_line = -1;
   PetscInt br_start_line = -1, br_end_line = -1;
+  PetscInt dcline_start_line = -1, dcline_end_line = -1;
   PetscInt gencost_start_line = -1, gencost_end_line = -1;
   PetscInt genfuel_start_line = -1, genfuel_end_line = -1;
   PetscInt loadcost_start_line = -1, loadcost_end_line = -1;
+
+  /* Number of lines for each data */
+  PetscInt bus_numlines, gen_numlines, br_numlines, dcline_numlines;
+  PetscInt load_numlines = 0;
   /* Number of blank lines in bus, gen, br, gencost, and genfuel branch arrays
    */
   PetscInt bus_nblank_lines = 0, gen_nblank_lines = 0;
-  PetscInt br_nblank_lines = 0;
+  PetscInt br_nblank_lines = 0, gencost_nblank_lines = 0;
+  PetscInt dcline_nblank_lines = 0;
+  PetscInt genfuel_nblank_lines = 0;
+  PetscInt loadcost_nblank_lines = 0;
```

Comment:
Similarly not sure if these variables actually end up being used anywhere

Reply:
Removed unused variables